android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -116,6 +121,20 @@ public void initNativeSdk(final ReadableMap rnOptions, Promise promise) {
             if (rnOptions.hasKey("sendDefaultPii")) {
                 options.setSendDefaultPii(rnOptions.getBoolean("sendDefaultPii"));
             }
+            if (rnOptions.hasKey("enableAutoPerformanceTracking") && rnOptions.getBoolean("enableAutoPerformanceTracking")) {
+                if (RNSentryModule.frameMetricsAggregator == null) {
+                    RNSentryModule.frameMetricsAggregator = new FrameMetricsAggregator();
```

Comment:
Should we always instantiate a new `frameMetricsAggregator` when we are going through the init code path?

Reply:
I guess we should. Originally had it this way as RN could call init multiple times if the JS bundle gets reloaded. However it makes sense to start fresh with the new aggregator + activity and let the garbage collector get the previous one.